.circleci/cimodel/data/pytorch_build_definitions.py

```diff
@@ -135,7 +135,9 @@ def gen_dependent_configs(xenial_parent_config):
 
     extra_parms = [
         (["multigpu"], "large"),
-        (["NO_AVX2"], "medium"),
+        (["YES_AVX512"], "medium"),
```

Comment:
Um, I don't think adding a `YES_` flag is the correct approach here, seeing as that's not the convention so far and having flags of different parity is confusing. Let's make AVX512 the default and only have NO_* flags?

Reply:
Can you suggest one then? We need one flag to enable the AVX512. Actually, the original setting only enables Default and AVX.

Reply:
Right now we have both a flag to enable AVX512 and a flag to disable AVX 512. Can you please explain the behavior currently?

Reply:
The logic is not that straightforward here, and I didn't intend to change it. You need to check the test.sh here.
```
if [[ "${BUILD_ENVIRONMENT}" == *-NO_AVX-* ]]; then
  export ATEN_CPU_CAPABILITY=default
elif [[ "${BUILD_ENVIRONMENT}" == *-NO_AVX2-* ]]; then
  export ATEN_CPU_CAPABILITY=avx
elif [[ "${BUILD_ENVIRONMENT}" == *-NO_AVX512-* ]]; then
  export ATEN_CPU_CAPABILITY=avx2
elif [[ "${BUILD_ENVIRONMENT}" == *-YES_AVX512-* ]]; then
  export ATEN_CPU_CAPABILITY=avx512
fi
```

We use the default if "--NO_AVX" appears in the environment string.
We use AVX if "--NO_AVX2-" appears at the head of environment string.
It means we always need one level up to support the current level.

Reply:
Those strings enforce an *upper bound* on the capability that's being run. By default we'll run the highest the machine supports. Thus, no YES_XXX variants were needed, since that's the default

Reply:
First, getting rid of YES_XXX is fine to me and I also tend to do that. The critical thing is that we need to be clear how the machine is allocated according to the string (I am checking with CircleCI support.). Basically, we want to a coverage of running the binary on AVX512, AVX2 and AVX, and Default CPUs.